www/scipylib/donations.rst

```diff
@@ -57,6 +57,223 @@ infrastructure:
 *This list is ordered by time (most recent contributions first) and was last
 updated in April 2019.*
 
+
+Thanks
+------
+
+SciPy is a community project sponsored by Enthought, Inc.
```

Comment:
That's not quite true anymore, and hasn't been for over a decade .....

Reply:
Ok, I'm mostly just taking the material here directly from the repo. I'll remove this line.